### PR TITLE
[MT-2963] Adding the capability for sending push notifications

### PR DESCRIPTION
**Pull request by JungHsuan**
### Changed:

1. Implement both v1 and the legacy FCM HTTP API for sending notifications.
2. Add buttons for sending two types of notifications: basic and carousel.
3. Update the README with instructions for sending notifications.

### CODE REVIEW

File: src/fcmHttpApi.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+const sendMessageLegacy = (fcmToken, messagePayload) => {
+  console.log('Send message by FCM Legacy HTTP API...');
+  const auth = `key=${legacy_server_key}`;
+  const body = {
+    to: fcmToken,
+    ...messagePayload,
+  };
+  return makePostRequest(LEGACY_API_URL, auth, body);
+};
+const sendMessageV1 = (fcmToken, messagePayload) => {
```
Comment (woodyl0813) [inline src/fcmHttpApi.js:25]: nit: spacing

**Review by woodyl0813 — APPROVED** [review]
Approved with static code review

**Comment by cedricshih** [comment]
nits: put settings into a single json file so the customers won't get confused, for example `app.json`:

```json
{
  "name": "AppierRNExample",
  "displayName": "AppierRNExample",
  "appier": {
    "appId": "<app ID>",
    "ios": {
      "appGroup": "<app group>",
      "isDev": "<APNS development environment>"
    },
    "fcm": {
      "serverKey": "<FCM server key>",
      "senderId": "<FCM sender ID"
    }
  }
}
```

File: App.js [hunk 2]
```diff
@@ -109,10 +116,10 @@ const App = () => {
           .then(result => console.log('result:', result))
           .catch(error => console.log('error:', error));
       }
-      const token = await messaging().getToken();
-      console.log('FCM token:', token);
-      if (token != null) {
-        setFcmToken(token);
+      const ftoken = await messaging().getToken();
```
Comment (cedricshih) [inline App.js]: nits: `ftoken` => `fcmToken`

**Comment by cedricshih** [comment]
nits: let's have some prompt messages when these settings are incorrect:

1. google-services.json
2. FCM server key

**Comment by cedricshih** [comment]
nits: let's have better campaign titles & messages, and use different titles and messages for different types of campaigns. for example:

- standard
  - title: We have the standard campaign! 🎉
  - message: Congrats! Now you can click to go back to the app.
- carousel
  - title: We have the carousel campaign! 🎉 
  - message: Congrats! You can click the right and left buttons to rotate the images.

**Review by cedricshih — APPROVED** [review]
Tested with Androud push.

**Pull request merged**